src/domain/session/room/RoomViewModel.js

```diff
@@ -197,18 +197,80 @@ export class RoomViewModel extends ViewModel {
         }
     }
     
+    async _getMessageInformations (message) {
+        let msgtype = "m.text";
+        if (message.startsWith("/")) {
+            let command=message.substring(1).split(" ");
```

Comment:
You can use [array destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#array_destructuring) here to make this a bit more readable:
```suggestion
            const [command, ...args] = message.substring(1).split(" ");
```
This way you don't have to use `command[0]` everywhere.